<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Dayanand-PC" ignored="0" name="Test1" tests="2" failures="1" timestamp="14 Apr 2017 15:07:19 GMT" time="26.742" errors="0">
  <testcase name="login_action" time="2.189" classname="com.Scripts.Login_script"/>
  <testcase name="create_user" time="0.398" classname="com.Scripts.User_script">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.page.User_List.newuser(User_List.java:48)
	at com.Scripts.User_script.create_user(User_script.java:30)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- create_user -->
</testsuite> <!-- Test1 -->
